tron/config/static_config.py

```diff
@@ -0,0 +1,23 @@
+from functools import partial
+
+import staticconf
+from staticconf import config
+
+FILENAME = "/nail/srv/configs/tron.yaml"
+NAMESPACE = "tron"
+
+
+def load_yaml_file():
+    staticconf.YamlConfiguration(filename=FILENAME, namespace=NAMESPACE)
+
+
+def build_configuration(filename, namespace):
```

Comment:
```suggestion
def build_configuration(filename: str, namespace: str) -> config.ConfigurationWatcher:
```
 
(could also maybe name this build_configuration_watcher(), but that's pretty nitpicky :p)